kafka-connectors/kafka-common/src/main/java/org/apache/apex/malhar/kafka/AbstractBaseKafkaInputOperator.java

```diff
@@ -77,12 +77,12 @@
  * @since 3.3.0
  */
 @InterfaceStability.Evolving
-public abstract class AbstractKafkaInputOperator implements InputOperator,
+public abstract class AbstractBaseKafkaInputOperator implements InputOperator,
```

Comment:
This change is needed.
Changed the class hierarchy from AbstractKafkaInputOperator <-- KafkaSinglePortInputOperator to
AbstractBaseKafkaInputOperator <-- AbstractKafkaInputOperator(malhar-kafka) <-- KafkaSinglePortInputOperator(malhar-kafka).
Please note, the functionality of AbstractKafkaInputOperator is same in both the hierarchies.
AbstractBaseKafkaInputOperator which is under malhar-common package contains createConsumer(...) abstract method.
AbstractKafkaInputOperator which is in malhar-kafka(supports 0.9 version) package creates the type of consumer based on the kafka version i.e, 0.9 consumer wrapper.